config/reports.json

```diff
@@ -298,20 +298,20 @@
 				]
 			}
 		},
-		"ttci": {
-			"name": "Time to Consistently Interactive",
+		"tti": {
+			"name": "Time to Interactive",
 			"type": "seconds",
 			"singular": "second",
-			"description": "The number of seconds from the time the navigation started until the page is fully interactive and the network is idle. This metric comes from [Lighthouse](https://developers.google.com/web/tools/lighthouse/audits/consistently-interactive) as is only available in mobile tests.",
+			"description": "The number of seconds from the time the navigation started until the page is fully interactive and the network is idle. This metric comes from [Lighthouse](https://developers.google.com/web/tools/lighthouse/audits/time-to-interactive) as is only available in mobile tests.",
```

Comment:
wdyt about replacing https://developers.google.com/web/tools/lighthouse/audits/time-to-interactive by https://docs.google.com/document/d/1GGiI9-7KeY3TPqS3YT271upUVimo-XiL5mwWorDUD4c ?

Reply:
Oh nice, didn't know about that doc. Can we link to both from each metric's description? I suggest we just append a new sentence, "Read more."